src/components/Footer/Footer.js

```diff
@@ -1,4 +1,171 @@
 import styled from 'styled-components';
+import {
+  faTwitter,
+  faFacebookF,
+  faInstagram,
+  faYoutube,
+} from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
 export default function Footer() {
-  return <div />;
+  return (
+    <Footerwrapper>
+      <Footertop>
+        <Innerwrap>
+          <Footertags>
+            {FOOTER_TAGS.map((tags, idx) => {
+              return (
+                <Footeratags key={idx}>
+                  <button href="/">{tags}</button>
+                </Footeratags>
+              );
+            })}
+          </Footertags>
+          <Findtheater>극장찾기</Findtheater>
+        </Innerwrap>
+      </Footertop>
+      <Footermain>
+        <Innerwrap>
+          <Footerci>
+            <img src="/images/logoGrey.jpeg" alt="로고" />
+          </Footerci>
+          <Footerinfo>
+            <p>서울 강남구 테헤란로 427 위워크 ARS 1234-1234</p>
+            <p>
+              대표자명 위코드 · 개인정보보호책임자 위코드 · 사업자등록번호
+              111-11-1111 · 통신판매업신고번호 제 1111-서울선릉-1234 호
+            </p>
+            <p>COPYRIGHT © Wegabox, Imc. All rights reserved</p>
+          </Footerinfo>
+          <Footermainicons>
+            <div>
+              <a href="#!">
+                <FontAwesomeIcon icon={faTwitter} />
+              </a>
+            </div>
+            <div>
+              <a href="#!">
+                <FontAwesomeIcon icon={faFacebookF} />
+              </a>
+            </div>
+
+            <div>
+              <a href="#!">
+                <FontAwesomeIcon icon={faInstagram} />
+              </a>
+            </div>
+            <div>
+              <a href="#!">
+                <FontAwesomeIcon icon={faYoutube} />
+              </a>
+            </div>
+          </Footermainicons>
+        </Innerwrap>
+      </Footermain>
+    </Footerwrapper>
+  );
 }
+
+const FOOTER_TAGS = [
+  '회사소개',
+  '인재채용',
+  '사회공헌',
+  '제휴/광고/부대사업문의',
+  '이용약관',
+  '위치기반서비스 이용약관',
+  '개인정보처리방침',
+  '윤리경영',
+];
+
+const Footerwrapper = styled.div`
+  display: block;
+  min-width: 1200px;
+  width: 100%;
+  height: 200px;
+  bottom: 0;
+  text-align: left;
+  background-color: #f8f8fa;
+
+  /* background-color: coral; */
+`;
```

Comment:
```suggestion
const FooterWrapper = styled.div`
  min-width: 1200px;
  height: 200px;
  text-align: left;
  background-color: #f8f8fa;
`;
```
- div는 원래 block입니다
- block이므로 width 100%이 불필요합니다
- relative, absolute, fixed 등이 아니므로 bottom 속성이 불필요합니다
- 불필요한 코드는 정리해주세요

Reply:
불필요한 코드와 속성들 삭제해주도록 하겠습니다 감사합니당